yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@electron/get@^2.0.2":
   version "2.0.2"
-  resolved "https://registry.yarnpkg.com/@electron/get/-/get-2.0.2.tgz#ae2a967b22075e9c25aaf00d5941cd79c21efd7e"
+  resolved "https://registry.npmjs.org/@electron/get/-/get-2.0.2.tgz"
```

Comment:
Note that a maintainer is going to have to revert the lockfile changes as well and re-apply them as per our dependency policy.

I'm not sure why all the URLs are being switched to `npmjs.org` here.

Reply:
Sure-- sorry, I just quickly slapped this PR up since it had just been sitting on my machine and I just wanted it to see the light of day. I had hacked up things a bit to install with `npm` and this is PR is just the logic extracted from that work. I can undo the whitespace changes and the syntax issue to make it work with Yarn, but I think someone on your side will need to re-lock things as you said.

Reply:
Done! Let me know if there's anything else that needs changed. I updated `p-limit` to the latest-- I had specifically chosen that old version because I was running into weird module type issues, but I was building `osx-sign` in an unorthodox manner so it might be alright to use the latest.